cfgov/unprocessed/js/modules/behavior/FlyoutMenu.js

```diff
@@ -162,8 +168,10 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
    */
   function _handleTriggerOver() {
     if ( !_touchTriggered && !_suspended ) {
-      this.dispatchEvent( 'triggerOver',
-                          { target: this, type: 'triggerOver' } );
+      this.dispatchEvent(
+        'triggerOver',
```

Comment:
L:184, 196, 229, 264 all have their first param on the same line as the function and should break to a new line like this one to keep consistent.